cpp/src/parquet/encoding.cc

```diff
@@ -861,8 +861,8 @@ std::shared_ptr<Buffer> ByteStreamSplitEncoder<DType>::FlushValues() {
   uint8_t* output_buffer_raw = output_buffer->mutable_data();
   const size_t num_values = values_.length();
   const uint8_t* raw_values = reinterpret_cast<const uint8_t*>(values_.data());
-#if defined(ARROW_HAVE_SSE4_2)
-  arrow::util::internal::ByteStreamSplitEncodeSSE2<T>(raw_values, num_values,
+#if defined(ARROW_HAVE_SIMD_SPLIT)
```

Comment:
sorry one more style comment.  We should consolidate this code into byte  byte_stream_split_.h so this code only calls ByteStreamSplitEncode.  Same with decode path below.

Reply:
done